perses/app/relative_point_mutation_setup.py

```diff
@@ -250,6 +250,23 @@ def __init__(self,
             validate_bool = False if old_res.name in ring_amino_acids or proposed_residue in ring_amino_acids else True
             new_positions, logp_proposal = geometry_engine.propose(topology_proposal, pos, beta,
                                                                    validate_energy_bookkeeping=validate_bool)
+
+            #check for charge change...
+            charge_diff = point_mutation_engine._get_charge_difference(current_resname = topology_proposal._old_topology.residue_topology.name,
+                                                                       new_resname = topology_proposal._new_topology.residue_topology.name)
+            if charge_diff != 0:
```

Comment:
Are fractional (but near zero) charges a concern? Or is `charge_diff` an integer?